cmd/xl-v1-healing-common.go

```diff
@@ -190,6 +217,7 @@ func disksWithAllParts(ctx context.Context, onlineDisks []StorageAPI, partsMetad
 			case isCorrupt:
 				fallthrough
 			case hErr == errFileNotFound, hErr == errVolumeNotFound:
+				logger.LogIf(ctx, hErr)
```

Comment:
doesn't need to be logged..

Reply:
if you need to log we should log in isCorrupt instead of `fileNotFound and VolumeNotFound`